kieker-extension-cassandra/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+	id 'java'
+	id 'java-library'
+}
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+// In this section you declare where to find the dependencies of your project
+repositories {
+	// Use jcenter for resolving your dependencies.
+	// You can declare any Maven/Ivy/file repository here.
+	jcenter()
+	maven {
+		url 'https://oss.sonatype.org/content/repositories/snapshots/'
+	}
+}
+
+dependencies {
+	api project(':kieker-monitoring')
+	api project(':kieker-analysis')
+
+	// This dependency is exported to consumers, that is to say found on their compile classpath.
+	//api 'net.kieker-monitoring:kieker:1.13-SNAPSHOT'
+
+	// This dependency is used internally, and not exposed to consumers on their own compile classpath.
+	implementation 'com.datastax.cassandra:cassandra-driver-core:3.4.0'
```

Comment:
Why this rather old version? Seems that 4.0.0 is currenty, and if this contains breaking API changes, 3.11.0 might be suitable

Reply:
Unfortunately, this implementation uses something from DataStax to access Cassandra. Thus, we have to stick with this library for the time being. However, I upgraded to com.datastax.cassandra:cassandra-driver-core:3.11.0